admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -372,5 +382,17 @@ module.exports = {
         },
       },
     }),
+    new HtmlWebpackPlugin({
+      filename: 'preload.tpl',
+      templateContent: '{{{preloadLinks}}}',
```

Comment:
```suggestion
      templateContent: '{{{preloadLinks}}}',
```

Any reason for the triple `{`?

Reply:
We could change it to anything else, but {{{}}} doesn't exist anywhere else, I find it pretty cool to see something else than any known syntax as a "template string"